spring/boot-autoconfigure/src/main/java/com/linecorp/armeria/spring/AnnotatedServiceRegistrationBean.java

```diff
@@ -130,4 +156,52 @@ public AnnotatedServiceRegistrationBean setDecorator(
         this.decorator = decorator;
         return this;
     }
+
+    /**
+     * Returns the exception handlers of the annotated service object.
+     */
+    public Collection<? extends ExceptionHandlerFunction> getExceptionHandlers() {
+        return exceptionHandlers;
+    }
+
+    /**
+     * Sets the exception handlers of the annotated service object.
+     */
+    public AnnotatedServiceRegistrationBean setExceptionHandlers(
+            @NotNull Collection<? extends ExceptionHandlerFunction> exceptionHandlers) {
```

Comment:
nit: Is `@NotNull` not necessary? Or, do we need to put `@NotNull` on the parameter of the other setter methods?

Reply:
Not necessary because all our classes are non-null by default, as annotated with `NonNullByDefault` at package level.

Reply:
Actually, this annotation is different from JSR-305. It seems to be used by Spring for validation. i.e. maybe necessary?

Reply:
Removed.